### PR TITLE
feat: introduce Caption component and styles

### PR DESCRIPTION
**Pull request by frshwtr**
## Describe your changes
* introduces Caption component as part of o3-typography
image

## Checklist before requesting a review

- [ ] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

File: components/o3-typography/src/css/tokens/internal/o3-typography/_variables.css [hunk 1]
```diff
@@ -85,4 +85,5 @@
   --_o3-typography-body-line-height: var(--o3-font-lineheight-1);
   --_o3-typography-body-color: var(--o3-color-palette-black);
   --_o3-typography-body-inverse-color: var(--o3-color-palette-white);
+  --_o3-typography-caption-color: #262a33;
```
Comment (ob6160) [inline components/o3-typography/src/css/tokens/internal/o3-typography/_variables.css:88]: is there a colour token for this?

File: components/o3-typography/src/tsx/caption.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {StyleArguments} from './getStyleAttributes';
+
+export const Caption: React.FC<
+	{children: React.ReactNode} & StyleArguments
+> = ({children, theme}) => {
```
Comment (ob6160) [inline components/o3-typography/src/tsx/caption.tsx:5]: we're pulling in the `theme` variable here but not using it

File: components/o3-typography/stories/story-templates.tsx [hunk 3]
```diff
@@ -2,6 +2,7 @@ import {Body, Heading, Link} from '../src/tsx';
 import {Emphasis, Strong, Subscript, Superscript} from '../src/tsx/body';
 import type {StoryObj} from '@storybook/react';
 import {BodyProps} from '@financial-times/o-typography/src/tsx/typography';
+import {Caption} from '../src/tsx/';
```
Comment (ob6160) [inline components/o3-typography/stories/story-templates.tsx:5]:
```suggestion
import {Caption} from '../src/tsx';
```

File: components/o3-typography/src/css/tokens/internal/o3-typography/_variables.css (hunk 1 above)
Comment (frshwtr) [reply components/o3-typography/src/css/tokens/internal/o3-typography/_variables.css:88]: > is there a colour token for this?

I couldn't find one in our pallette. I can raise with Mary to see if it's suitable in pallette :)

**Review by ob6160 — APPROVED** [review]

**Review by akomiqaia — APPROVED** [review]

File: components/o3-typography/src/css/tokens/internal/o3-typography/_variables.css [hunk 4]
```diff
@@ -85,4 +85,5 @@
   --_o3-typography-body-line-height: var(--o3-font-lineheight-1);
   --_o3-typography-body-color: var(--o3-color-palette-black);
   --_o3-typography-body-inverse-color: var(--o3-color-palette-white);
+  --_o3-typography-caption-color: var(--o3-color-palette-slate);
```
Comment (akomiqaia) [inline components/o3-typography/src/css/tokens/internal/o3-typography/_variables.css:88]: This seems like black-80?

**Review by akomiqaia — COMMENTED** [review]
DO we need to update readme with `caption` component information?

**Pull request merged**